public/main/inc/lib/sessionmanager.lib.php

```diff
@@ -8269,15 +8269,119 @@ public static function getGridColumns(
                     ];
                 }
                 break;
+
+            case 'simple':
+                $columns = [
+                    '#',
+                    get_lang('Name'),
+                    get_lang('Category'),
+                    get_lang('SessionDisplayStartDate'),
```

Comment:
Use the English version of text: 
Example: use "Start date to display" instead of SessionDisplayStartDate.

Check the English version in the trad4all.inc.php file

Reply:
The same for the other variables.